grammar/vernacextend.mlp

```diff
@@ -82,7 +82,7 @@ let make_clause_classifier cg s { r_patt = pt; r_class = c; } =
          "classifiers. Only one classifier is called.") ^ "\n");
     (make_patt pt,
       ploc_vala None,
-      <:expr< fun () -> (Vernacexpr.VtUnknown, Vernacexpr.VtNow) >>)
+      <:expr< fun () -> (Vernacexpr.VtSideff [], Vernacexpr.VtNow) >>)
```

Comment:
I think you should really turn this case `None, None` into an error.
Turning a command with no given classification into a `VtSideff` is wrong, and I hope people removed this looong warning from their code already (they had 8.5 and 8.6 to do that).